<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
border-radius: 15px
</string>
    </property>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(223, 255, 181,0.7);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="horizontalSlider_Progress">
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal 
{ 
	border: 1px solid #999999; 
	height: 2px; 
	margin: 0px 0;
	left: 5px; 
	right: 5px; 
}

QSlider::handle:horizontal
{
	border: 0px ;
	border-image: url(:/img/picsilder.png);			/*图片自行添加*/
	width: 17px; 
	margin: -8px -8px -8px -8px; 
}

QSlider::add-page:horizontal
{
	background-color: #cbcbcb; 
}

QSlider::sub-page:horizontal
{
	background-color: #CC0033;
};


</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0);</string>
        </property>
        <property name="text">
         <string>-04：48</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_volume">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/pic/sound.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Previous">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/pic/rewind-button.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_play">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/pic/play.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_next">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/pic/fast-forward-button.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_music_name">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;楷体&quot;;
</string>
        </property>
        <property name="text">
         <string>知我</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_author">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgba(0);
color: rgb(255, 255, 255);
font: pt &quot;楷体&quot;;
</string>
        </property>
        <property name="text">
         <string>-国风堂/哦漏</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>428</x>
       <y>12</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QSlider::groove:vertical 
{ 
	background: #cbcbcb;
	width: 6px;
	border-radius: 1px;
	padding-left:-1px;
	padding-right:-1px;
	padding-top:-1px;
	padding-bottom:-1px; 
}

QSlider::add-page:vertical
 { 

	
	background-color:  #CC0033;
	width: 10px; 
	border-radius: 2px; 
}
QSlider::handle:vertical 
{
	border-image:url(:/img/pic/silder.png);
	margin: -2px -7px -2px -7px; 
	height: 17px; 
};
background-color: rgba(255, 255, 255,0); 
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
